axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
Yeah I've thought about it more during my walk and agree (on both counts).

On Tue, 8 Sep 2015 15:13 Owen Campbell <notifications@github.com> wrote:

> In axelrod/tests/unit/test_player.py
> <https://github.com/Axelrod-Python/Axelrod/pull/293#discussion_r38929857>:
>
> > @@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
> >
> >      name = "Player"
>
> I think the TestPlayer class needs to loop over an expected_behaviour
> dictionary and, for each key in that dictionary, check that it exists in
> the player under test and that the values match. That will cope with
> existing tests which don't have all the dimensions specified.
>
> But I also think that should be a separate ticket.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/293/files#r38929857>.
>